src/kotlin_script.cpp

```diff
@@ -60,13 +61,13 @@ ScriptInstance* KotlinScript::_instance_create(const Variant** p_args, int p_arg
         KotlinBindingManager::get_instance_binding(p_this);
     }
 
-    KtClass* kt_class {get_kotlin_class()};
 #ifdef DEBUG_ENABLED
-    LOG_VERBOSE(vformat("Try to create %s instance.", kt_class->resource_path));
+    JVM_CRASH_COND_MSG(!is_valid(), vformat("An invalid script was attempted to be used. Make sure you have properly built your project."));
```

Comment:
Log which script it was. This would help debugging later in case there is an issue